app/helpers/application_helper.rb

```diff
@@ -4,9 +4,10 @@ module ApplicationHelper
   include Pagy::Frontend
 
   def nice_date_times(datetime, options = {})
-    return "" if datetime.nil?
+    return '' if datetime.nil?
+
     datetime = datetime.in_time_zone(Rails.application.config.display_timezone) if options[:localtime] && Rails.application.config.display_timezone
-    "#{datetime.strftime('%a')} #{datetime.day.ordinalize} #{datetime.strftime('%b %Y %H:%M')}"
+    I18n.l(datetime, format: "%a #{datetime.day.ordinalize} %b %Y %H:%M")
```

Comment:
hi @ldodds @urbanwide quick q - can we concatenate `nice_dates` and `nice_times_only` here?  
I may be wrong but looks like the same `: es_full` formatting is used 
e.g. `"#{nice_dates(datetime)} #{nice_times_only(datetime)}"`

Reply:
@iant did you resolve this with Leigh? Otherwise I don't mind taking a look tomorrow (sorry have been away!)

Reply:
hi @urbanwide thanks yep all good now. i've pushed a small update. thanks 🙏